static/js/components/request.vue

```diff
@@ -128,8 +128,8 @@ export default {
       }
     }, 300),
     instrumentToDataType: function(value){
-      if(!_.isEmpty(this.available_instruments) && value) {
-        return this.available_instruments[value].type;
+      if (value.includes('NRES') || value.includes('FLOYDS')) {
+        return 'SPECTRA';
       }
       return 'IMAGE';
```

Comment:
This should probably just be an if/else

Reply:
It effectively is an if/else. Unless it's better to have if() {} else {}

Reply:
Yes let's be explicit with an else in case this method is modified later.